apps/api/v2/src/modules/billing/services/billing.service.ts

```diff
@@ -118,7 +126,7 @@ export class BillingService implements OnModuleDestroy {
     return url;
   }
 
-  async setSubscriptionForTeam(teamId: number, subscriptionId: string, plan: PlatformPlan) {
+  async setSubscriptionForTeam(teamId: number, subscriptionId: string, plan: PlatformPlan, priceId?: string) {
```

Comment:
Optional parameters indicate a function should be 2 functions. Can we create a new function `setPerActiveUserSubscriptionForTeam` and have `priceId: string` required string and then use it and rename `setSubscriptionForTeam` to `setPerBookingSubscriptionForTeam`.